apisix/cli/ngx_tpl.lua

```diff
@@ -84,6 +84,16 @@ stream {
     lua_ssl_trusted_certificate {* ssl.ssl_trusted_certificate *};
     {% end %}
 
+    # for stream logs
+    {% if stream.enable_access_log == false then %}
+    access_log off;
+    {% else %}
+    log_format main escape={* stream.access_log_format_escape *} '{* stream.access_log_format *}';
+
+    access_log {* stream.access_log *} main buffer=16384 flush=3;
+    {% end %}
+    open_log_file_cache max=1000 inactive=60;
```

Comment:
We don't need it, as the log file name doesn't contain variable. There is only one open log file.
See https://nginx.org/en/docs/http/ngx_http_log_module.html#open_log_file_cache

Reply:
Ack. Thanks